Project/Assets/Prairie/Framework/Script/Interaction/FirstPersonInteractor.cs

```diff
@@ -83,6 +93,26 @@ private void drawToolbar(List<Annotation> annotations)
 	{
 		// TODO: Draw a preview (and input button) for each value in `annotations`
 
+		if (annotations.Count != 0)
+		{
+			float xpos = 0.1f * Screen.width;
+			float ypos = 0.7f * Screen.height;
+			int button = 0;
+
+			// Make a background box
+			GUI.Box(new Rect (xpos, ypos, 250, 120), "Area Annotations");
```

Comment:
The margins should of the box should be static. The size of the box should be relative based on the size of the screen.

As it stands, on a giant monitor, you have a big gap on the left side of the screen, and a relatively small toolbar. On smaller screens, you get a small margin and a toolbar which covers the scene.

The best would be to have static margins, and sizes based on the screen size _but with minimums and maximums_.